docs/src/components/graph-page/graph-page.js

```diff
@@ -359,17 +360,28 @@ const GraphPage = (props) => {
       </div>
       <Modal show={show} onHide={() => setShow(false)} contentClassName="graph-modal">
         <Modal.Header closeButton>
-          <Modal.Title style={{fontFamily: "Raleway"}}>Image Title</Modal.Title>
+          <Modal.Title style={{fontFamily: "Raleway"}}>{descriptions[activeImage - 1].title} by {descriptions[activeImage - 1].artist} - {generatedImages.includes(activeImage) ? "Generated" : "Original"}</Modal.Title>
         </Modal.Header>
         <Modal.Body>
+          <img
+            src={require(`../../images/pieces/${activeImage}.jpg`)}
+            alt={activeImage}
+          />
+          <h3 className="graph-title">Image Description</h3>
```

Comment:
Can you add some padding/margin here, more space between the image and the Image Description section